src/test/java/com/wikia/webdriver/common/core/configuration/Configuration.java

```diff
@@ -17,45 +9,58 @@
 import java.util.logging.Level;
 import java.util.logging.Logger;
 
+import lombok.Getter;
+
+import org.apache.commons.lang.StringUtils;
+import org.openqa.selenium.Dimension;
+import org.openqa.selenium.WebDriverException;
+import org.yaml.snakeyaml.Yaml;
+
+import com.wikia.webdriver.common.core.exceptions.TestEnvInitFailedException;
+import com.wikia.webdriver.common.properties.Credentials;
+
 /**
  * Configuration handler. This Class should handle run configuration and global properties.
- * Configuration handling: <ol> <li>Look for the property key in testConfig map, if key is present,
- * return the value</li> <li>Look for the property key in system properties - return value of this
- * property if key present </li> <li>If no System Property is found - value is provided from
- * configuration files (config_default.yml and config.yml). Values provided in config.yml, are
- * overriding values from config_default.yml</li> </ol>
+ * Configuration handling:
+ * <ol>
+ * <li>Look for the property key in testConfig map, if key is present, return the value</li>
+ * <li>Look for the property key in system properties - return value of this property if key present
+ * </li>
+ * <li>If no System Property is found - value is provided from configuration files
+ * (config_default.yml and config.yml). Values provided in config.yml, are overriding values from
+ * config_default.yml</li>
+ * </ol>
  */
 public class Configuration {
 
   private static final String DEFAULT_CONFIG_FILE_NAME = "config_default.yml";
   private static final String LOCAL_CONFIG_FILE_NAME = "config.yml";
   private static final Logger LOGGER = Logger.getLogger(Configuration.class.getName());
-  private static String wikiaDomain;
   private static String geoEdgeCountry;
   private static Map<String, String> defaultConfig;
   private static Map<String, String> testConfig = new HashMap<>();
+  @Getter(lazy = true)
+  private static final String wikiaDomain = getEnvType().getWikiaDomain();
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Rename this constant name to match the regular expression '^[A-Z][A-Z0-9]*(_[A-Z0-9]+)*$'. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://sonar-s1/coding_rules#rule_key=squid%3AS00115)